lib/vehicle.rb

```diff
@@ -22,4 +30,16 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def plate_type
+    if self.electric_vehicle?
+      :ev 
+    elsif self.antique?
```

Comment:
```suggestion
    if electric_vehicle?
      :ev 
    elsif antique?
```
While not functionally incorrect, we don't need to use `self` here. Ruby automatically will perform these methods on the object on which they were called.